.renovaterc.json

```diff
@@ -2,12 +2,24 @@
   "extends": [
     "config:base",
     ":pinOnlyDevDependencies",
-    ":prHourlyLimitNone",
-    ":prConcurrentLimitNone",
-    ":automergeMinor",
```

Comment:
I think we should keep this one, as we use the [renovate-approve bot](https://github.com/renovatebot/renovate-approve-bot) to automatically approve then merge all non-major deps, and it should then also work with lock file maintenance. 

However, you guys have also experimented with another bot for auto-mergers, correct?

Reply:
> you guys have also experimented with another bot for auto-mergers

Not really, we have `kodiakhq` which we triggers an auto merge only when we assign a specific label.

> I think we should keep this one

Sure, I can add that back

Reply:
I see, then everything looks great methinks. Thanks for thinking of us when aligning all the repos!